pandas/tests/frame/methods/test_reindex.py

```diff
@@ -1097,12 +1097,11 @@ def test_reindex_with_categoricalindex(self):
                 df2.reindex(["a", "b"])
 
         # args NotImplemented ATM
-        msg = r"argument {} is not implemented for CategoricalIndex\.reindex"
-        with pytest.raises(NotImplementedError, match=msg.format("method")):
+        msg = "method pad not yet implemented for CategoricalIndex"
+        with pytest.raises(NotImplementedError, match=msg):
             df.reindex(["a"], method="ffill")
-        with pytest.raises(NotImplementedError, match=msg.format("level")):
-            df.reindex(["a"], level=1)
-        with pytest.raises(NotImplementedError, match=msg.format("limit")):
+        msg = "limit argument only valid if doing pad, backfill or nearest reindexing"
```

Comment:
i think this is inaccurate, since limit is _never_ valid

Reply:
This has been reverted.